Added map component in Treatmentcenter page

sanjays442 opened the pull request:
Added map component in Treatmentcenter page

sanjays442 committed:
Integrated Map component in Treatment center page

sanjays442 committed:
Integrated the map component to Treatmentcenter page

sanjays442 committed:
Merge branch 'master' into treatcenterMapWithComponent

ravenzheng commented on app/treatmentcenterMap/treatment-center-map.html:
```diff
@@ -215,7 +215,7 @@
                             <div id="mapwrapper">
                                 <div id="map_base">
-                                    <search-by-state-treatment></search-by-state-treatment>
+                                    <state-Map></state-Map>
```
When you use components in html, use dash connected and lower case form please.

ravenzheng commented on app/treatmentcenterMap/index.js:
```diff
@@ -5,13 +5,11 @@ var angular = require('angular'),
       template: htmlTemplate,
-      controller: 'TreatmentcenterMapCtrl'
-    });
+      controller: 'TreatmentcenterMapCtrl',
```
Please remove the extra comma.

ravenzheng commented on app/treatmentcenterMap/ctrl.js:
```diff
@@ -42,7 +42,7 @@ function getarrayValue(url) {
     }
 }
-function ctrl($scope, $routeParams, service) {
+function ctrl($scope, $routeParams, service, $location) {
```
What does this "$location"  here? If you want to use it, then avoid usage of " window.location" in the following code.

ravenzheng commented on app/home/map/index.js:
```diff
@@ -1,54 +1,57 @@
-    var target = event.target;
-    target.setAttribute('fill', config.overcolor);
-  });
+    var _ctrl = this;
```
Please use two spaces to replace a tab.

ravenzheng reviewed (changes requested):
Please write a more meaningful commit message. Here is a recommended blog on how to write a good commit message. [The art of the commit](https://alistapart.com/article/the-art-of-the-commit)

sanjays442 committed:
Removed the extra comma

Removed the extra comma from addiction_network_frontend/app/treatmentcenterMap/index.js on line number 11

sanjays442 committed:
Removed the variable $location as it was not used in this controller

Removed the variable $location from addiction_network_frontend/app/treatmentcenterMap/ctrl.js as it was not used in this controller
